proto/_package_info.py

```diff
@@ -37,7 +37,12 @@ def compile(name, attrs):
     proto_module = getattr(module, "__protobuf__", object())
 
     # A package should be present; get the marshal from there.
-    package = getattr(proto_module, "package", module_name)
+    # When package is empty, upb based protobuf fails with an
```

Comment:
Please mark all hacks like this with a TODO saying it should be removed once the new version of protobuf with fixed upb is out.

Reply:
Done.